pkg/os/exec.go

```diff
@@ -30,6 +29,15 @@ func run(command string, args []string, env map[string]string) (string, string,
 	return stdOut.String(), stdErr.String(), err
 }
 
+func run(command string, args []string, env map[string]string) (string, string, error) {
+	logging.Debugf("Running '%s %s'", command, strings.Join(args, " "))
+	return runCmd(command, args, env)
+}
+
+func runPrivate(command string, args []string, env map[string]string) (string, string, error) {
+	return runCmd(command, args, env)
+}
```

Comment:
Maybe you can add a `private` arg to `run` rather than having multiple one-line helpers? This would be closer to what is done in `ssh.go` (merely a suggestion).
We need a log similar to `logging.Debugf("About to run SSH command with hidden output")` though in the `runPrivate` case.

Reply:
@cfergeau this is what I thought first but then we need to change the `run` function signature everywhere to add that private arg value to false where we don't want it. I did thought about it and I think abstraction make it bit better then changing the signature everywhere.

Reply:
> We need a log similar to logging.Debugf("About to run SSH command with hidden output") though in the runPrivate case.

Good point, will update this accordingly.